marketdepthviewer/marketdepthviewer.go

```diff
@@ -162,10 +131,10 @@ func subscribeToMarketData(dataclient api.TradingDataServiceClient) {
 	if err != nil {
 		log.Panicln("Unable to send event bus request on the stream", err)
```

Comment:
I guess since it's a tool and if some of these errors are not recoverable, it might be ok to panic/fatal. But for consistency it's best if we bubble all the errs up and handle them there. Not super important though